migration/migration-technical-details.md

```diff
@@ -66,14 +83,16 @@ During this phase, validators send a transaction signaling their readiness. Unli
+#### Transition Block
+
+The `GenesisConfig` is generated after the candidate block is mined. If the pre-registered validators run the tool before the activation window starts, the tool will have already migrated most of the history by the time the candidate block is mined. After the candidate block is mined and the network reaches 80% readiness, the tool migrates the final part of the history and generates the "genesis" block, which becomes the transition block for the PoS chain.
 
-The activation tool activates at the candidate block - the block height aligned with an election block corresponding to the PoS chain’s structure. At this point, the tool scans and analyzes transactions sent to the burn address. The transition starts if at least 80% of the total stake is ready.
+The activation tool activates at the candidate block - the block height aligned with an election block corresponding to the PoS chain’s structure. At this point, the tool scans and analyzes transactions sent to the network. The transition starts if at least 80% of the total stake is ready.
 
-If the transition does not succeed within the initial activation window, a new one is opened until the 80% readiness target is achieved. The new activation window starts at the block immediately following the candidate block. Each activation window spans 1440 PoW blocks, equivalent to one day.
+#### Failing 80% of Readiness
 
-Once the 80% total stake readiness requirement is reached, the candidate block becomes the transition block. The activation tool captures the state of the PoW chain at this block, including all accounts, the first validator list, allocated stake, and transaction history, and generates the genesis block for the PoS chain. This block marks the beginning of the Nimiq PoS blockchain.
+If the transition does not succeed within the initial activation window, a new one opens immediately. Each new window starts at the block following the first 24 hours and spans 1 440 PoW blocks (24 hours), repeating until the 80% readiness target is achieved.
 
 ## Nimiq PoS
 
-To ensure the integrity of this process, Nimiq suggests a waiting period of 10 confirmations before the PoS chain officially begins. This precaution is taken into consideration for potential block rebranching.
+Once 80% readiness is achieved and the final portion of the history is migrated, the transition block is finalized, marking the official beginning of the PoS chain. All PoW blocks mined after the transition block are disregarded as if they never occurred. From this point forward, the network operates entirely under the PoS consensus mechanism.
```

Comment:
Once 80% readiness is achieved, the PoS chain begins. Any subsequent transaction that is included in the PoW chain after the candidate block, is not considered as part of the PoS chain.